### PR TITLE
Django 2.2 compatibility

### PR DESCRIPTION
**Pull request by urtzai**

### CODE REVIEW

File: django_messages/models.py [hunk 1]
```diff
@@ -79,7 +79,7 @@ def __str__(self):
 
     def get_absolute_url(self):
         return ('messages_detail', [self.id])
```
Comment (arneb) [inline django_messages/models.py]: shouldn't we change this row to `return reverse('messages_detail', [self.id])` when we remove the decorator?

File: django_messages/models.py [hunk 2]
```diff
@@ -51,7 +51,7 @@ class Message(models.Model):
     """
     subject = models.CharField(_("Subject"), max_length=140)
     body = models.TextField(_("Body"))
-    sender = models.ForeignKey(AUTH_USER_MODEL, related_name='sent_messages', verbose_name=_("Sender"), on_delete=models.SET_NULL)
+    sender = models.ForeignKey(AUTH_USER_MODEL, related_name='sent_messages', verbose_name=_("Sender"), on_delete=models.PROTECT)
```
Comment (arneb) [inline django_messages/models.py]: I think I like the PROTECT rule better than SET_NULL!
Can we add a migration?

**Comment by arneb** [comment]
Thanks for the PR. I added two comments. I'm willing to merge this, but please give me your opinion on the comments first.

File: django_messages/models.py (hunk 1 above)
Comment (urtzai) [reply django_messages/models.py]: > return reverse('messages_detail', [self.id]

Yep, definitely, we need to return a value with reverse, if not we will get an error

Comment (urtzai) [reply django_messages/models.py]: > return reverse('messages_detail', [self.id]

Yep, definitely, we need to return a value with reverse, if not we will get an error

File: django_messages/models.py (hunk 2 above)
Comment (urtzai) [reply django_messages/models.py]: Yes, I forgot that the sender can't be null! Sorry!!

**Comment by vonmaster** [comment]
Is there any update on this PR? Do you need any help?

**Comment by arneb** [comment]
@vonmaster the PR still needs the change mentioned above 

> shouldn't we change this row to return reverse('messages_detail', [self.id]) when we remove the decorator?

**Pull request merged**